Lib/test/test_pdb.py

```diff
@@ -4199,6 +4200,38 @@ def f(x):
             self.assertIn('42', stdout)
             self.assertIn('return x + 1', stdout)
 
+    def test_zipimport(self):
+        with os_helper.temp_dir() as temp_dir:
+            os.mkdir(os.path.join(temp_dir, 'source'))
+            zipmodule = textwrap.dedent(
+                """
+                def bar(x):
+                    return x + 1
+                """
+            )
+            script = textwrap.dedent(
+                f"""
+                import sys; sys.path.insert(0, {repr(os.path.join(temp_dir, 'zipmodule.zip'))})
+                import foo
+                foo.bar(41)
+                """
+            )
+
+            with zipfile.ZipFile(os.path.join(temp_dir, 'zipmodule.zip'), 'w') as zf:
+                zf.writestr('foo.py', zipmodule)
+            with open(os.path.join(temp_dir, 'script.py'), 'w') as f:
+                f.write(script)
+
+            stdout, _ = self._run_pdb([os.path.join(temp_dir, 'script.py')], '\n'.join([
+                'n',
+                'n',
+                'b foo.bar',
+                'c',
+                'p x + x',
+                'q'
+            ]))
+            self.assertIn('82', stdout)
```

Comment:
What is '82'?

Reply:
It's the result of `p x + x`, I want to confirm that we did stop inside `foo.bar`. I can maybe change the testing code to use string so it's more obvious? Or I can add a comment here. The other possibility is to use a more advanced command to rule out the false positives (for example, you can't simply check `bar` because it will be shown anyway).

Reply:
Yes, a string would be better.

Reply:
I constructed a small piece of unique string and the assertion is close to the command - I think that made it clearer.